masq/src/commands/scan_command.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::command_context::CommandContext;
+use crate::commands::commands_common::{transaction, Command, CommandError};
+use clap::{App, Arg, SubCommand};
+use masq_lib::messages::{ScanType, UiScanRequest, UiScanResponse};
+use std::fmt::Debug;
+use std::str::FromStr;
+
+pub const SCAN_COMMAND_TIMEOUT_MILLIS: u64 = 10000;
```

Comment:
The scan command blocks until the scan is complete. Ten seconds is a guess at how long a scan should take. I don't have any hard data for it.

Reply:
Yeah, I got it. I was just thinking if we could get into a hopeless waiting because something had gone wrong...having to wait so long then...but I don't think it's a real threat. So ignore that comment, please.